build.gradle

```
@@ -1,33 +1,14 @@
-import io.github.wzieba.tracks.plugin.BuildTimePlugin
 import io.github.wzieba.tracks.plugin.TracksExtension
-import io.gitlab.arturbosch.detekt.Detekt
 
-buildscript {
-    ext.kotlinVersion = '1.5.31'
-    ext.navigationVersion = '2.4.0-beta02'
-    ext.daggerVersion = '2.40.1'
-    ext.wordPressLoginVersion = '0.10.0'
-    ext.detektVersion = '1.18.1'
-
-    repositories {
-        google()
-        gradlePluginPortal()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath 'com.automattic.android:fetchstyle:1.1'
-        classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion"
-        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navigationVersion"
-        classpath "com.google.dagger:hilt-android-gradle-plugin:$daggerVersion"
-        classpath "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:$detektVersion"
-        classpath "io.github.wzieba.tracks:io.github.wzieba.tracks.plugin:1.0.0"
-    }
+plugins {
+    id 'com.automattic.android.fetchstyle'
+    id 'io.gitlab.arturbosch.detekt'
+    id 'io.github.wzieba.tracks.plugin'
+    id 'io.sentry.android.gradle'
+    id 'dagger.hilt.android.plugin' apply false
+    id 'androidx.navigation.safeargs.kotlin' apply false
 }
```

We have a new warning `Warning: Please apply google-services plugin at the bottom of the build file.` during configuration phase. This can be reproduced with `./gradlew --rerun-tasks`. It's a similar issue to the ones discussed above and applying the following suggestion seems to be addressing the warning. Once again, I think this is a lifecycle/dependency issue, but I can't be entirely sure without going into their source code and I think this is a safe enough change to make without spending the extra time on it.

```suggestion
    id 'com.google.gms.google-services' apply false
}
```

---

Thanks! I agree, addressed in 10cd44fa935d904299587aa00c6eaea95b88f5d2

I've seen this warning before but I didn't know that this will fix that. I felt that even with this warning it's safe to move `google-services` to Plugin DSL as this is even the recommended way in [their README](https://github.com/google/play-services-plugins/tree/master/google-services-plugin).